File: myblog-server/src/main/java/com/site/blog/constants/property/MyBlog.java
package com.site.blog.constants.property;

import lombok.Data;
import org.springframework.boot.context.properties.ConfigurationProperties;
import org.springframework.stereotype.Component;

/**
 * @author yanni
 * @date time 2022/6/16 2:54
 * @modified By:
 */
@Component
@Data
@ConfigurationProperties(value = "myblog")
public class MyBlog{
    private String version;
    private String name;
}

File: blog-sever/server-jpa/src/main/kotlin/com/site/blog/constants/property/MyBlog.kt
package com.site.blog.constants.property

import org.springframework.boot.context.properties.ConfigurationProperties
import org.springframework.stereotype.Component

/**
 * @author yanni
 * @date time 2022/6/16 2:54
 * @modified By:
 */
@Component
@ConfigurationProperties(value = "myblog")
data class MyBlog(
    var version: String? = null,
    var name: String? = null
)